conf/locale/babel_underscore.cfg

```diff
@@ -1,9 +1,5 @@
 # Use this configuration file for extracting strings from .underscore files.
-[underscore: cms/templates/**.underscore]
-input_encoding = utf-8
-[underscore: common/templates/**.underscore]
-input_encoding = utf-8
-[underscore: lms/templates/**.underscore]
+[underscore: **.underscore]
```

Comment:
I don't understand how that can be the case, but if you are both fine with it then I am too.